cypress/features/regression/buttonAsSiblingIconPart2.feature

```diff
@@ -0,0 +1,72 @@
+Feature: Button as a sibling component
+  I want to check Button as a sibling component properties
+
+  Background: Open Button as a sibling component page default
+    Given I open "Button" component page as sibling in no iframe
+
+  @positive
+  Scenario Outline: Set Button icon to <iconType>
+    When I check has icon checkbox
```

Comment:
`When I check has icon checkbox` should be moved to the Background

Reply:
fix